drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -12,6 +12,20 @@ function DragAndDropBlock(runtime, element, configuration) {
     var state = undefined;
     var __vdom = virtualDom.h();  // blank virtual DOM
 
+    // Keyboard accessibility
+    var CTRL = 17;
+    var ESC = 27;
+    var RET = 13;
+    var SPC = 32;
+    var TAB = 9;
+    var M = 77;
+    var QUESTION_MARK = 63;
+
+    var ctrlDown = false;
```

Comment:
@itsjeyd I don't think I understand what this `CtrlDown` does. If it is there to check for "Crtl+Something", there's [better way](http://stackoverflow.com/questions/2767126/how-to-detect-that-ctrlr-was-pressed) (basically, `e.ctrlKey`) to do that than global variable.